Cargo.toml

```diff
@@ -104,4 +104,4 @@ harness = false
 
 [patch.crates-io]
 # TODO(XXX): Remove this once rcgen has cut a release w/ CRL support included.
-rcgen = { git = 'https://github.com/est31/rcgen.git' }
+rcgen = { git = 'https://github.com/est31/rcgen.git', rev = '83e548a06848d923eada1ac66d1a912735b67e79' }
```

Comment:
Thanks, I think we can probably get a rcgen tag Soon :tm: but this is a good fix until then. In retrospect I should have included a SHA from the get-go.